rs/nns/governance/canister/governance.did

```diff
@@ -577,6 +626,62 @@ type Neuron = record {
   transfer : opt NeuronStakeTransfer;
   known_neuron_data : opt KnownNeuronData;
   spawn_at_timestamp_seconds : opt nat64;
+
+  // When a neuron is first created, this is set to create_timestamp_seconds.
+  //
+  // This is part of the "periodic confirmation" feature, which was proposed and
+  // (overwhelmingly) adopted in NNS motion proposal 132411:
+  // https://dashboard.internetcomputer.org/proposal/132411
+  //
+  // This gets updated to the current time when the user performs related
+  // actions, as described in the aforementioned proposal.
+  //
+  // For neurons created prior to this feature, this field will be set to
+  // 2024-09-01T00:00:00 UTC.
+  //
+  // Per NNS policy, this is opt. Nevertheless, it will never be null (unless we
+  // later deprecate this feature).
```

Comment:
```suggestion
  // Per NNS policy, this is opt. Nevertheless, it will never be null.
```

I wondered if this is more confusing than helping (as people might wonder whether we already think about deprecating the feature again, although I know this is now what you meant)

Reply:
Yes, this whole thing could be confusing, and is not strictly necessary.

I guess you already figured out the motivation here, but let me just say for the record, unless you are familiar with the whole API (not good to assume the reader has such familiarity), it will probably seem weird that this is opt, because based on everything said earlier, it can be inferred fairly easily that this is never null. Therefore, when the reader finally reaches the point where it says `opt`, they are likely to think "Wait, why is this opt?? Is there some situation that I didn't notice where this could be null????". This attempts to preempt that question by saying (something like), "no, you did not miss anything. You are right to think this can never be null. Here is why the field is opt anyway.".

Reply:
No I agree and I like that you explain that. I was more refering to the part "unless we laterdeprecate this feature" which adds a condition that may or may not happen but they may think "why do we deprecate this". 
Or did you mean something else?

Reply:
I see. Yes, that adds another layer of possible confusion. I don't have a strong opinion -> I'll just go with what you suggested.

Reply:
Done.